crewai-agentic-flow-demo/test_basic.py

```
@@ -0,0 +1,174 @@
+
+# Add src to path
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), 'src'))
+
+def test_tool_mock():
+    """Test the Firecrawl tool mock functionality"""
+    print("Testing Firecrawl tool mock...")
+    
+    # We'll test just the mock search functionality without CrewAI dependencies
+    try:
+        # Import just the mock search logic
+        from crewai_agentic_flow.tools.firecrawl_tool import FirecrawlSearchTool
```

Ruff (0.12.2)

`crewai_agentic_flow.tools.firecrawl_tool.FirecrawlSearchTool` imported but unused

Remove unused import: `crewai_agentic_flow.tools.firecrawl_tool.FirecrawlSearchTool`

(F401)